src/s3ql/backends/gs.py

```diff
@@ -53,6 +53,14 @@ def __init__(self, options):
         if self.use_oauth2:
             self.hdr_prefix = 'x-goog-'
 
+        self.google_app_credentials = None
+        if self.password == '':
+            log.info('Password empty, using google application default account')
```

Comment:
The objects will be instantiated more than once, it's ugly to have it log this every time. Please either change to `log.debug`, or add a `LOG_ONCE` stanza.